pkg/ebpftracer/c/headers/common/buffer.h

```diff
@@ -12,6 +12,14 @@
 #include <common/context.h>
 #include <common/network.h>
 
+/* BPF_FUNC_bpf_ringbuf_commit, BPF_FUNC_bpf_ringbuf_discard, and
+ * BPF_FUNC_bpf_ringbuf_output flags.
+ */
+enum {
```

Comment:
This should not be needed btw, `bpf.h` already defines those
https://elixir.bootlin.com/linux/v5.10/source/include/uapi/linux/bpf.h#L4029